pkg/domain/infra/abi/images_list.go

```diff
@@ -30,12 +30,16 @@ func (ir *ImageEngine) List(ctx context.Context, opts entities.ImageListOptions)
 		for j, d := range img.Digests() {
 			digests[j] = string(d)
 		}
+		isDangling, err := img.IsDangling(ctx)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error checking if image %q is dangling", img.ID())
```

Comment:
NIT: I don't think we should start errors with "error", Podman prepends Error: on all Errors, I don't want to end up with 
Error: error checking if image ...

Reply:
I'll tackle that in a separate PR.  There more such cases in the code that I can tackle at once.